### DISCUSSION TITLE
wallet support

### DISCUSSION THREAD

**Discussion by lacostaguy** [Chia-Network/chia-blockchain#10748] [Q&A]
I see where some older posts asked about cold wallet storage options, (like Ledger) were being "worked on".  any update here? or does anyone know where i can store my chia tokens.  I need to get off an exchange, and would like to use web3 wallet or my ledger.  thanks

**Comment by brianwfreeman** [comment]
No Ledger support yet. 
Have you heard of Nucle.io?  I don't personally use it, but it's out there.
I store my XCH in a cold wallet.  Offline machine generated seed, funds stored in that seed to HODL.

**Reply by lacostaguy** [reply]
Thanks Brian. Which cold wallet do you use?

**Comment by brianwfreeman** [comment]
I make one with the Chia client on an offline computer.

On Wed, Mar 16, 2022 at 11:17 AM lacostaguy ***@***.***>
wrote:

> Thanks Brian. Which cold wallet do you use?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Chia-Network/chia-blockchain/discussions/10748#discussioncomment-2374984>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEI6LRMAEFWGSLMYHW4NVMTVAIQTPANCNFSM5QZTTZ5Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you commented.Message ID:
> <Chia-Network/chia-blockchain/repo-discussions/10748/comments/2374984@
> github.com>
>
-- 



Brian W. Freeman

Attorney at Law

Certified Specialist in CA Workers' Compensation Law



http://WorkersCompensationLawyerInc.com
<http://workerscompensationlawyerinc.com/>

Phone & Fax: (844) 4 AVG JOE

**Reply by lacostaguy** [reply]
Cool.. lmk if you'd like to chat off-line sometime. I'm in North San Diego area.. would love to compare notes and always looking to connect with other Chia/crypto fans

**Reply by lacostaguy** [reply]
Hey Brian, I'm thinking about just sending my Chia to the Chia wallet. It's on a computer that doesn't stay on full time, so I won't be able to take advantage of the staking. But seems like the best, safest option. Comments?

**Comment by brianwfreeman** [comment]
There is no such thing as staking.  Chia wallet works best IMO.

**Comment by lacostaguy** [comment]
Thx.  Sorry, I meant “farming” .. lol
